packages/loot-core/src/types/models/dashboard.d.ts

```diff
@@ -37,6 +37,7 @@ export type CashFlowWidget = AbstractWidget<
     conditions?: RuleConditionEntity[];
     conditionsOp?: 'and' | 'or';
     timeFrame?: TimeFrame;
+    isSimpleView?: boolean;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** we could make this more readable + open for future by using an enum.
```suggestion
    mode?: 'condensed' | 'full';
```